spec/src/main/asciidoc/configuration.adoc

```diff
@@ -0,0 +1,307 @@
+//
+// Copyright (c) 2023 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+[[sec:configuration-properties]]
+== Configuration
+
+OpenTelemetry MUST be configured by MicroProfile Config following the semantics of configuration properties detailed in https://github.com/open-telemetry/opentelemetry-java/tree/v{otel-java-version}/sdk-extensions/autoconfigure[OpenTelemetry SDK Autoconfigure {otel-java-version}].
```

Comment:
OpenTelemetry MUST be configured by MicroProfile Config based the semantics of configuration properties detailed in https://github.com/open-telemetry/opentelemetry-java/tree/v{otel-java-version}/sdk-extensions/autoconfigure[OpenTelemetry SDK Autoconfigure {otel-java-version}].

Property names MAY be prefixed like `prefix.otel.property`. Prefixes MUST not colide with existing OpenTelemetry properties and the property names in the suffix part MUST match the ones defined in the OpenTelemetry SDK Autoconfigure.

---
Motivation:
We need to add the possibility of implementation specific prefixes. 
Property names like `prefix.otel.property`, should be allowed. Example: `quarkus.otel.sdk.disabled`. 
Implementations might have a dedicated namespace and might want to preserve that, also property value validation get's better contained.

Reply:
I think the point of standard is to have one universal set of properties, they are already prefixed -- with `otel.`. Implementations are free to provide additional means.

Reply:
I don't think we should update the otel properties here but just provide a better way to configure them.

Reply:
A configuration strategy can be implemented to pass non prefixed otel properties to the system in order to pass the TCK